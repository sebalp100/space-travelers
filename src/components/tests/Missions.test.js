import * as React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/configureStore';
import Mission from '../Missions';

test('First Snapshot test', () => {
  const component = renderer
    .create(
      <Provider store={store}>
        <Mission />
        ;
      </Provider>,
    )
    .toJSON();

  expect(component).toMatchSnapshot();
});
