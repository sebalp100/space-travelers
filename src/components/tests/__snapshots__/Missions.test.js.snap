// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First Snapshot test 1`] = `
Array [
  <div />,
  ";",
]
`;
